tools/ApiDocsGenerator/release-notes/rn-json-schema.md

```diff
@@ -4,29 +4,59 @@ title: JsonSchema.Net
 icon: fas fa-tag
 order: "09.01"
 ---
-# [5.5.1](https://github.com/gregsdennis/json-everything/pull/616) {#release-pointer-5.5.1}
+# [6.0.0](https://github.com/gregsdennis/json-everything/pull/619) {#release-schema-6.0.0}
+
+Updated for multi-framework support with .Net Standard 2.0 and .Net 8 with Native AOT support, including updating _System.Text.Json_ from v6 to v8.
+
+Most of the changes to support Native AOT involve either updating internal implementation or creating overloads that do.  Whatever could not be updated was marked as requiring reflection, which will generate warnings when building Native AOT applications.
+
+## Breaking changes
+
+- Dropping support for .Net Standard 3.1 - May still be used, but exact behavior cannot be guaranteed. 
+- Keyword converters are no longer responsible for writing their property name when serializing.  This is now handled by `JsonSchema`'s converter.
+- `ErrorMessages.Dependencies` removed.  Was previously marked obsolete.  Use `.DependentSchemas` and `.DependentRequired` when customizing the message for `dependencies`.
+- `ErrorMessages.InvalidItemsForm` removed.  Invalid form of `items` results in an exception.
+- `ErrorMessages.InvalidPattern` removed.  Invalid patterns result in an exception.
+- `JsonSchemaBuilderExtensions.Validate()` removed.  Was previously marked obsolete.
+- `JsonSchemaExtensions.Validate()` and overloads removed.  Were previously marked obsolete.
+
+## Additions
+
+- `JsonSchema.TypeInfoResolver` to expose all of the type resolvers contained in the library.  Can be used to create a combined `SerializationOptions` by using a `Json.More.TypeResolverOptionsManager` in your `JsonSerializerContext`.
+- `DependsOnAnnotationsFromAttribute<T>` (non-generic version still exists)
+
+## Other chanages
+
+- `JsonSchema.FromFile(string, JsonSerializerOptions?)` marked AOT-incompatible and replaced with `.FromFile(string)`.
```

Comment:
I wouldn't characterize as "replaced". FromText & friends that take an `options` can be AOT compatible, same as JsonSerializer.Deserialize, the caller just needs to ensure that the SerializerOptions includes JsonSchema.TypeInfoResolver to guarantee it will work. Then they can suppress the warnings.

Reply:
I'll reword